ui/src/components/review/emissions-widget/emissions-widget.tsx

```diff
@@ -243,8 +238,42 @@ const EmissionsWidget: FunctionComponent<Props> = (props) => {
     }
   }
 
+  // handle download notification
   
-
+  const handleDownload = (actorID: string, fileType: string) => {
+      setToggleMenu(false);
+      setOpen(true);
+      setSeverity("info");
+      setMessage('Prepering your dataset');
+      setnotificationBackgroundColor('#001EA7');
+      setDownloadError(false);
+    fetch(`${ServerUrls.api}/v1/download/${actorID}-emissions.${fileType}`)
+      .then(res=> {
+        return res.blob();
+      })
+      .then(blob=>{
+        const downloadLink = document.createElement('a');
```

Comment:
Maybe it makes sense to remove this element afterwards? I guess it has no text content so won't show up in the rendered page, but doing some housekeeping might be good.